e2e/bitrise.yml

```diff
@@ -0,0 +1,260 @@
+      - BITRISE_PROJECT_PATH: sample-apps-ios-workspace-swift.xcworkspace
+      - BITRISE_SCHEME: sample-apps-ios-workspace-swift
+      - SIMULATOR_OS_VERSION: "latest"
+      - OUTPUT_TOOL: xcpretty
+      - OUTPUT_DIR: $BITRISE_DEPLOY_DIR
+      - BITRISE_APP_DIR_PATH_EXPECTED: "$BITRISE_DEPLOY_DIR/sample-apps-ios-workspace-swift.app"
+      - BITRISE_APP_DIR_PATH_LIST_EXPECTED: "$BITRISE_DEPLOY_DIR/sample-apps-ios-workspace-swift.app"
+      - CODE_SIGNING_ALLOWED: "no"
+    after_run:
+      - _common
+
+  test_multi_target_workspace_custom_derived_data_path:
+    envs:
+      - XCODEBUILD_OPTIONS: "-derivedDataPath ./ddata"
+      - SAMPLE_APP_URL: https://github.com/bitrise-samples/sample-apps-ios-workspace-swift.git
+      - BRANCH: watch
+      - BITRISE_PROJECT_PATH: sample-apps-ios-workspace-swift.xcworkspace
+      - BITRISE_SCHEME: sample-apps-ios-workspace-swift
+      - SIMULATOR_OS_VERSION: "latest"
+      - OUTPUT_TOOL: xcodebuild
+      - OUTPUT_DIR: $BITRISE_DEPLOY_DIR
+      - BITRISE_APP_DIR_PATH_EXPECTED: "$BITRISE_DEPLOY_DIR/sample-apps-ios-workspace-swift.app"
+      - BITRISE_APP_DIR_PATH_LIST_EXPECTED: "$BITRISE_DEPLOY_DIR/sample-apps-ios-workspace-swift.app|$BITRISE_DEPLOY_DIR/bitfall.sample-apps-ios-workspace-swift-watch.app"
+      - CODE_SIGNING_ALLOWED: "no"
+    after_run:
+      - _common
+
+  test_multi_target_workspace:
+    envs:
+      - XCODEBUILD_OPTIONS:
+      - SAMPLE_APP_URL: https://github.com/bitrise-samples/sample-apps-ios-workspace-swift.git
+      - BRANCH: watch
+      - BITRISE_PROJECT_PATH: sample-apps-ios-workspace-swift.xcworkspace
+      - BITRISE_SCHEME: sample-apps-ios-workspace-swift
+      - SIMULATOR_OS_VERSION: "latest"
+      - OUTPUT_TOOL: xcpretty
+      - OUTPUT_DIR: $BITRISE_DEPLOY_DIR
+      - BITRISE_APP_DIR_PATH_EXPECTED: "$BITRISE_DEPLOY_DIR/sample-apps-ios-workspace-swift.app"
+      - BITRISE_APP_DIR_PATH_LIST_EXPECTED: "$BITRISE_DEPLOY_DIR/sample-apps-ios-workspace-swift.app|$BITRISE_DEPLOY_DIR/bitfall.sample-apps-ios-workspace-swift-watch.app"
+      - CODE_SIGNING_ALLOWED: "no"
+    after_run:
+      - _common
+
+  test_multi_target_workspace_code_signing_allowed:
+    envs:
+      - XCODEBUILD_OPTIONS:
+      - SAMPLE_APP_URL: https://github.com/bitrise-samples/sample-apps-ios-workspace-swift.git
+      - BRANCH: watch
+      - BITRISE_PROJECT_PATH: sample-apps-ios-workspace-swift.xcworkspace
+      - BITRISE_SCHEME: sample-apps-ios-workspace-swift
+      - SIMULATOR_OS_VERSION: "latest"
+      - OUTPUT_TOOL: xcpretty
+      - OUTPUT_DIR: $BITRISE_DEPLOY_DIR
+      - BITRISE_APP_DIR_PATH_EXPECTED: "$BITRISE_DEPLOY_DIR/sample-apps-ios-workspace-swift.app"
+      - BITRISE_APP_DIR_PATH_LIST_EXPECTED: "$BITRISE_DEPLOY_DIR/sample-apps-ios-workspace-swift.app|$BITRISE_DEPLOY_DIR/bitfall.sample-apps-ios-workspace-swift-watch.app"
+      - CODE_SIGNING_ALLOWED: "yes"
+    after_run:
+      - _common
+
+  # React Native tests are disabled due to detox installation failure
```

Comment:
should we just delete these?

Reply:
Good question. I'd say yes, but I don't have enough context to decide. @ofalvai @godrei thoughts?

Reply:
- [ ] Delete unused tests